### DISCUSSION TITLE
EPA emission factors and uncertainty analysis

### DISCUSSION THREAD

**Discussion by MelaniaMassarotto** [USEPA/supply-chain-factors#19] [Q&A] [answered]
I used some EPA emission factors for a GHG inventory project. I need to calculate the uncertainty analysis, but I'm note sure how to treat the uncertainty for the EPA emission factors. Could you please help me?

**Comment by WesIngwersen** [comment, marked as answer]
Great question.  Unfortunately the factors themselves do not have associated variances.  However, an approach I would recommend is selecting different factors from the larger set of supply chain factor that could be relevant for your purchased good of interest -- these could come from closely related goods and services, factors from other years, factors based on other model types (summary vs detail; industry vs. commodity) and then use all those factors to compute your emissions. Then you can calculate some variance information from your range of estimated emissions.
